static/app/components/onboarding/footerWithViewSampleErrorButton.tsx

```diff
@@ -0,0 +1,397 @@
+  ]);
+
+  const handleSkipOnboarding = useCallback(() => {
+    if (!projectId) {
+      return;
+    }
+
+    if (onboardingContext.data[projectId].status !== OnboardingStatus.WAITING) {
+      return;
+    }
+
+    trackAdvancedAnalyticsEvent('growth.onboarding_clicked_skip', {
+      organization,
+      source: 'targeted_onboarding_first_event_footer',
+    });
+
+    const selectedProjectId = selectedProject?.id;
+
+    let pathname = `/organizations/${organization.slug}/issues/?`;
+    if (selectedProjectId) {
+      pathname += `project=${selectedProjectId}&`;
+    }
+
+    openChangeRouteModal({
+      router,
+      nextLocation: {
+        ...router.location,
+        pathname: (pathname += `referrer=onboarding-first-event-footer-skip`),
+      },
+      setClientState,
+      clientState,
+    });
+  }, [
+    router,
+    organization,
+    setClientState,
+    clientState,
+    selectedProject,
+    onboardingContext,
+    projectId,
+  ]);
+
+  const handleViewError = useCallback(() => {
+    if (!projectId) {
+      return;
+    }
+
+    trackAdvancedAnalyticsEvent('onboarding.view_error_button_clicked', {
+      organization,
+      new_organization: !!newOrg,
+      project_slug: projectSlug,
+    });
+
+    if (clientState) {
+      setClientState({
+        ...clientState,
+        state: 'finished',
+      });
+    }
+
+    router.push({
+      ...router.location,
+      pathname: `/organizations/${organization.slug}/issues/${onboardingContext.data[projectId].firstIssueId}/?referrer=onboarding-first-event-footer`,
+    });
+  }, [
+    organization,
+    newOrg,
+    router,
+    clientState,
+    setClientState,
+    onboardingContext,
+    projectId,
+    projectSlug,
+  ]);
+
+  return (
+    <Wrapper newOrg={!!newOrg} sidebarCollapsed={!!preferences.collapsed}>
+      <Column>
+        {projectData?.status === OnboardingStatus.WAITING && newOrg && (
+          <Button onClick={handleSkipOnboarding} priority="link">
+            {t('Skip Onboarding')}
+          </Button>
+        )}
+      </Column>
+      <StatusesColumn>
+        {projectData?.status === OnboardingStatus.WAITING ? (
+          <WaitingForErrorStatus>
+            <IconCircle size="sm" />
+            {t('Waiting for error')}
+          </WaitingForErrorStatus>
+        ) : projectData?.status === OnboardingStatus.PROCESSED ? (
+          <ErrorProcessedStatus>
+            <IconCheckmark isCircled size="sm" color="green300" />
+            {t('Error Processed!')}
+          </ErrorProcessedStatus>
+        ) : (
+          <ErrorProcessingStatus>
+            <RefreshIcon size="sm" />
+            {t('Processing error')}
+          </ErrorProcessingStatus>
+        )}
+      </StatusesColumn>
+      <ActionsColumn>
+        {projectData?.status === OnboardingStatus.PROCESSED ? (
+          <Button priority="primary" onClick={handleViewError}>
+            {t('View Error')}
+          </Button>
+        ) : (
+          <CreateSampleEventButton
```

Comment:
this file is just a copy of the `footer.tsx`  and the only diff is this "View Sample Error" button